### PR TITLE
#160451563 User should be able to rate an article

### PR DESCRIPTION
**Pull request by Mrshinezee**
#### What does this PR do?
A user should be able to rate an article

#### Description of Task to be completed?
implement a functionality that allows user rate an article

#### How should this be manually tested?
- start the app
- select an article to view
- on the view article page, a rating icon will be available
- click on the icon to rate

#### What are the relevant pivotal tracker stories?
[#160451571](https://www.pivotaltracker.com/story/show/160451563, "Users should be able to rate articles")

#### Screenshots (if appropriate)
screen shot 2018-11-21 at 3 56 43 pm

### CODE REVIEW

File: src/components/reusables/reaction/Rating.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// react libraries
+import React from 'react';
+
+// third-party libraries
+import PropTypes from 'prop-types';
+import Rating from 'react-rating';
+import RadialProgress from '../radialProgress/RadialProgress';
```
Comment (e0works) [inline src/components/reusables/reaction/Rating.jsx:9]: RadialProgress should be commented as 
```js
// components
import RadialProgress from '../radialProgress/RadialProgress';
```

File: src/components/article/viewArticle/ViewArticlePage.jsx [hunk 2]
```diff
@@ -57,24 +57,28 @@ class ViewArticle extends Component {
...
+      id,
       title,
       imageUrl,
-      body
+      body,
+      slug,
+      rating,
+      reactions
     } = this.props.publishedArticle.Articles;
     const shareUrl = `https://lotus-ah-staging.herokuapp.com/api/v1${this.props.location.pathname}`;
```
Comment (e0works) [inline src/components/article/viewArticle/ViewArticlePage.jsx]: please change this line

```js
const shareUrl = `https://lotus-ah-staging.herokuapp.com/api/v1${this.props.location.pathname}`;
```
to 
```js
const shareUrl = `https://lotus-ah-frontend.herokuapp.com/api/v1${this.props.location.pathname}`;
```

File: src/reducers/article/__test__/rateArticle.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          payload: failMessage
+        })
+      ).toEqual(
+        {
+          userRating: failMessage
+        }
+      );
+    });
+  });
+
```
Comment (e0works) [inline src/reducers/article/__test__/rateArticle.test.js]: please add a new line here

**Review by e0works — CHANGES_REQUESTED** [review]
Nice job @Mrshinezee

**Review by koiic — APPROVED** [review]
Looks good to me

**Review by e0works — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
@Mrshinezee 
Rebase.

**Pull request merged**